app/Http/Controllers/Installer/DatabaseController.php

```diff
@@ -37,4 +29,86 @@ public function database()
                          ->with(['fails' =>$ex->getMessage()]);
         }
     }
+
+    /**
+     * Migrate and seed the database.
+     *
+     * @return array
+     */
+    public function migrateAndSeed()
+    {
+        $outputLog = new BufferedOutput;
+
+        $this->sqlite($outputLog);
+
+        return $this->migrate($outputLog);
+    }
+
+    /**
+     * Run the migration and call the seeder.
+     *
+     * @param \Symfony\Component\Console\Output\BufferedOutput $outputLog
+     * @return array
+     */
+    private function migrate(BufferedOutput $outputLog)
+    {
+        try {
+            Artisan::call('migrate', ['--force'=> true], $outputLog);
+        } catch (Exception $e) {
+            throw new Exception($e->getMessage());
+        }
+
+        return $this->seed($outputLog);
+    }
+
+    /**
+     * Seed the database.
+     *
+     * @param \Symfony\Component\Console\Output\BufferedOutput $outputLog
+     * @return array
+     */
+    private function seed(BufferedOutput $outputLog)
+    {
+        try {
+            Artisan::call('db:seed', ['--force' => true], $outputLog);
+        } catch (Exception $e) {
+            throw new Exception($e->getMessage());
+        }
+
+        return $this->response(trans('installer_messages.final.finished'), 'success', $outputLog);
+    }
+
+    /**
+     * Return a formatted error messages.
+     *
+     * @param string $message
+     * @param string $status
+     * @param \Symfony\Component\Console\Output\BufferedOutput $outputLog
+     * @return array
+     */
+    private function response($message, $status, BufferedOutput $outputLog)
+    {
+        return [
+            'status' => $status,
+            'message' => $message,
+            'dbOutputLog' => $outputLog->fetch(),
+        ];
+    }
+
+    /**
+     * Check database type. If SQLite, then create the database file.
+     *
+     * @param \Symfony\Component\Console\Output\BufferedOutput $outputLog
+     */
+    private function sqlite(BufferedOutput $outputLog)
```

Comment:
Reason for SqlLite database connection?

Reply:
Was part of package. Commented it for now if we need it in future